index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Portfolio</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <header>
+            <nav class="nav_container">
+                <div class="logo">
+                     <img src="images/MyLogo.png" alt="Logo">
+                </div>
+     
+                <div class="toggle">
+                    <img src="images/Union.png" alt="Logo">
+                </div>
```

Comment:
- [ ] Please consider wrapping the toggle with a `nav` tag instead of a `div` tag as the toggle which is known as the hamburger menu consists of navigation links and also to improve the semantic structure of your HTML😄. You can read more on [SEMANTIC HTML.](https://www.w3schools.com/html/html5_semantic_elements.asp) Also please rename the `alt` text as the toggle is not `logo`. See my suggestions below.

```suggestion
                <nav class="toggle">
                    <img src="images/Union.png" alt="hamburger menu">
                </nav>
```